compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.9'
+
+services:
+  frontend:
+    build:
+      context: client
+    image: frontend:0.1
+    container_name: frontend
+    ports:
+      - 5173:5173
+    environment:
+      PUBLIC_GATEWAY_BASE_URL: http://gateway:8000/
+
+  gateway:
+    build:
+      context: backend/gateway
+    image: gateway:0.1
+    container_name: gateway
+    ports:
+      - 8000:8000
+    environment:
+      PORT: 8000
+      JWT_SECRET: zeusemployeemanagementjwtsecret
+      JWT_EXPIRES: 60
+      JWT_IGNORE_EXPIRATION: true
+      EMPLOYEES_URL: http://profile:8010
+      FRONTEND_URL: http://frontend:5173
+
+  profile:
+    build:
+      context: backend/profile
+    image: profile:0.1
+    container_name: profile
+    ports:
+      - 8010:8010
+    environment:
+      PORT: 8010
+      MONGODB_URI: mongodb+srv://ozddev:GzNZ3FzNgy8hNmMi@zemcluster1.dz4rzdu.mongodb.net/employee?retryWrites=true&w=majority
```

Comment:
I don't know if its possible or not, but can you somehow take this string from the .env so its not exposed? Or maybe add the local connection string as default `mongodb://localhost:27017`

Reply:
Also this repo is public, and other can see this connection string now

Reply:
Also you can use this official image for a new local mongo docker compose service https://hub.docker.com/_/mongo

Reply:
I have changed it to localhost for now